core/utils/arp.h

```diff
@@ -8,33 +8,37 @@
 namespace bess {
 namespace utils {
 
-// A basic ARP header definition.
-struct[[gnu::packed]] ARP {
+// A basic ARP header definition
+struct[[gnu::packed]] Arp {
   // Ethernet hardware format for hrd
-  static const be16_t kEtherHardwareFormat = 1;
-
+  enum HardwareAddress : uint16_t {
+    kEthernet = 1,
+  };
+  
   enum Opcode : uint16_t {
-    REQUEST = 1,
-    REPLY = 2,
-    REVREQUEST = 3,
-    REVREPLY = 4,
-    INVREQUEST = 8,
-    INVREPLY = 8,
+    kRequest = 1,
+    kReply = 2,
+    kRevRequest = 3,
+    kRevReply = 4,
+    kInvRequest = 8,
+    kInvReply = 9,
   };
 
-  be16_t hrd;   // format of hardware address
-  be16_t pro;   // format of protocol address
-  uint8_t hln;  // length of hardware address
-  uint8_t pln;  // length of protocol address
-  be16_t op;    // ARP opcode (command)
+  be16_t hw_address;            // format of hardware address (hrd)
+  be16_t proto_address;         // format of protocol address (pro)
+  uint8_t hw_address_length;    // length of hardware address (hln)
+  uint8_t proto_address_length; // length of protocol address (pln)
+  be16_t opcode;                // ARP opcode (command) (op)
 
   // ARP Data
-  Ethernet::Address sha;  // sender hardware address
-  be32_t sip;             // sender IP address
-  Ethernet::Address tha;  // target hardware address
-  be32_t tip;             // target IP address
+  Ethernet::Address sender_hw_addr;  // sender hardware address (sha)
+  be32_t sender_ip_addr;             // sender IP address (sip)
+  Ethernet::Address target_hw_addr;  // target hardware address (tha)
+  be32_t target_ip_address;          // target IP address (tip)
```

Comment:
`target_ip_addr`..? (to be consistent with `sender_ip_addr`)

Otherwise, LGTM.